cnd/src/db/load_swaps.rs

```diff
@@ -50,50 +92,11 @@ struct BitcoinEthereumBitcoinEtherAcceptedSwap {
     at: NaiveDateTime,
```

Comment:
This should have been renamed to `start_of_swap` during the resume swap PR.  I added it to my TODO list :)